test/e2e/serial/tests/configuration.go

```diff
@@ -221,7 +221,7 @@ var _ = Describe("[serial][disruptive][slow] numaresources configuration managem
 					}
 				}
 				return true, nil
-			}).WithTimeout(10 * time.Minute).WithPolling(30 * time.Second).Should(BeTrue())
+			}).WithTimeout(2 * timeout).WithPolling(30 * time.Second).Should(BeTrue())
```

Comment:
the problem here is we are introducing a coupling between all the tests which use the timeout variable.
Suppose we need to increase the timeout for a single test for thatever reason. Perhapse the specific test is doing something that takes longer to complete.
This will mean all the tests that use the variable will increase the timeout. Is it correct? is it *more* readable?
Another reason: why it's easier/more readable to reason in terms of block of `timeouts` unit?
IOW, why is `2 * timeout` better than `10 * time.Minute`?